apps/nextjs/.storybook/decorators/ChatStoreDecorator.tsx

```diff
@@ -10,8 +10,9 @@ import {
 declare module "@storybook/csf" {
   interface Parameters {
     chatStoreState?: {
-      ailaStreamingStatus: AilaStreamingStatus;
+      ailaStreamingStatus?: AilaStreamingStatus;
       queuedUserAction?: string | null;
+      isStreaming?: boolean;
```

Comment:
Have a look at the screenshot I've sent in slack for typing this a bit nicer with `Partial<ChatStore>`